dist/backdraft.js

```diff
@@ -1275,8 +1275,19 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var configsArray = this.columnsConfig;
+      var cleanConfigs = this.columnsConfig.map(function(a,b,c){
+        if (b===0) {
+          return a
+        }
+        if(a.attr&&a.visible){
+          return a}
+        }).filter(function(n){
+           return n != undefined
```

Comment:
surprised jslint didn't squawk (do we not have it on backdraft?) but this should be `!==`.. or do we want `typeof n !== "undefined"` ?